spec/controllers/api/users_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe Api::UsersController do
+  describe '#index' do
+    let!(:user_1) { FactoryGirl.create(:user) }
+    let!(:user_2) { FactoryGirl.create(:user) }
+    let(:result) { JSON.parse(response.body) }
+    before do
+      get :index
+    end
+
+    it 'returns valid json' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.